test/utils/combineReducers.spec.js

```diff
@@ -159,16 +159,14 @@ describe('Utils', () => {
               return state + 1
             case 'decrement':
               return state - 1
-            // Never do this in your code:
-            case ActionTypes.INIT:
-              return 0
             default:
-              return undefined
```

Comment:
Can we keep the test that if you forget a `default` case you'll get this warning? It's an important thing to remember to check.